### DISCUSSION TITLE
Considering making a full release of v0.10.0, thoughts?

### DISCUSSION THREAD

**Discussion by coaxial** [benjaminoakes/maid#296] [General]
I've been using v0.10.0-alpha for several weeks now and haven't run into any issues on Ruby 3.2.

Is it time to make a full release of v0.10.0 so that new users will get this version without specifying `--pre`, or are there issues I'm not aware of that should be fixed before that happens?

**Comment by benjaminoakes** [comment]
Let's go for it.  People have started to reach out about not being able to install Maid, so the upsides would seem to outweigh the downsides.

**Comment by coaxial** [comment]
Well, this was laborious but here we are: https://github.com/maid/maid/pull/303

The lesson learnt is that it is imperative for `release-please` that an empty commit is made **directly on `master`** and not via a PR for it to make the release.

I have updated the wiki accordingly: https://github.com/maid/maid/wiki/Release-Process#to-create-a-full-release-after-a-series-of-prereleases

**Reply by benjaminoakes** [reply]
Thanks! That's an unfortunate restriction as many projects disallow commits directly to master.  Glad we have it figured out, though.

**Reply by coaxial** [reply]
Yes, I had to change the setting that admins can bypass the branch protections. I'm trusting us not to abuse so that's not too big a deal, but it does require extra caution before `git push`, to make sure we're not on `master`!
